sycl/include/sycl/ext/oneapi/matrix/static-query-use.hpp

```diff
@@ -152,12 +152,12 @@ struct tpu_params<tpu::amx, Ta, Tb, Tc, 0, 0, 0,
 
   template <typename Group>
   using joint_matrix_a =
-      joint_matrix<Ta, defaultM, defaultK, use::a, layout::row_major, Group>;
+      joint_matrix<Ta, use::a, defaultM, defaultK, layout::row_major, Group>;
   template <typename Group>
   using joint_matrix_b =
-      joint_matrix<Tb, defaultK, defaultN, use::b, layout::packed_b, Group>;
+      joint_matrix<Tb, use::b, defaultK, defaultN, layout::packed, Group>;
   template <typename Group>
-  using joint_matrix_c = joint_matrix<Tc, defaultM, defaultN, use::accumulator,
+  using joint_matrix_c = joint_matrix<Tc, use::accumulator, defaultM, defaultN,
                                       layout::row_major, Group>;
 
```

Comment:
tpu_params will have to take layout of A and layout of B as template arguments. Then you pass them here.
Layout if the accumulator matrix should be layout::dynamic